sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -19,6 +19,152 @@
 #include <set>
 #include <vector>
 
+/// \defgroup sycl_graph DPC++ Execution Graph
+///
+/// SYCL, unlike OpenCL, provides a programming model in which the user doesn't
+/// need to manage dependencies between kernels and memory explicitly. The DPC++
+/// Runtime must ensure correct execution with respect to the order commands are
+/// submitted.
+///
+/// This document describes the part of the DPC++ Runtime that is responsible
+/// for building and processing dependency graph.
+///
+/// ## A couple of words about DPC++ and SYCL execution and memory model
+///
+/// The SYCL framework defines command group (\ref CG) as an entity that
+/// represents minimal execution block. The command group is submitted to SYCL
+/// queue and consists of a kernel or an explicit memory operation, and their
+/// requirements. The SYCL queue defines the device and context using which the
+/// kernel should be executed.
+///
+/// The commands that contain explicit memory operations include copy, fill,
+/// update_host and other operations. It's up to implementation how to define
```

Comment:
```suggestion
/// update_host and other operations. It's up to implementation how to implement
```
??

Reply:
I wonder if repetition in such cases is a stylistic mistake...